2. flow_control/Q15.c

```diff
@@ -1 +1,26 @@
 // Write a program to read an array of n integers and print the array in reverse order using a for loop.
+#include<stdio.h>
+int main(){
+    int n,array[100],i;
+    printf("Enter the number of elements in the array = ");
+    scanf("%d", &n);
+
+    for ( i = 0; i <= n-1; i++) 
```

Comment:
second condition in for loop can be written like this i<n for brevity

Reply:
printf("\nreverse array : ");
    for ( i = n-1; i >= 0; i--)
    {
        printf("%d ", array[i]);
    }
 What to use in place of n-1 in this situation?

Reply:
no need, its correct here.